source-code/core/src/utilities/polyfills/util.ts

```diff
@@ -0,0 +1,468 @@
+// @ts-nocheck
```

Comment:
this file creates is the polyfill to be able to log an object in a pretty way instead of `'[object Object]'`